docs/developers.md

```diff
@@ -15,15 +15,14 @@ Helm.
 We use Make to build our programs. The simplest way to get started is:
 
 ```console
-$ make bootstrap build
+$ make build
 ```
 
 NOTE: This will fail if not running from the path `$GOPATH/src/helm.sh/helm`. The
```

Comment:
Should this note be updated because I think by using Go modules `GOPATH` is no longer mandatory? If so then update in install doc as well.